### ISSUE TITLE
Could not find module in path: 'bem-ts'

### ISSUE THREAD

**Issue by weaintplastic** [ybiquitous/bem-ts#892]
When upgrading from 0.12.1 to 0.13.x typscript is complaining about the missing module `bem-ts`

```
ModuleNotFoundError: Could not find module in path: 'bem-ts' relative to '/src/index.ts'
```

I've created a code sandbox to demonstrate the issue running the following code

```ts

import bem from "bem-ts";

const b = bem("block");

console.log(b());
```

bem-ts v0.12.1: https://codesandbox.io/s/typescript-forked-959orq?file=/src/index.ts
bem-ts v0.13.0: https://codesandbox.io/s/typescript-forked-slo8ef?file=/src/index.ts

I hope the description makes sense and you can follow my lead. Happy to support fixing this bug.

**Comment by ybiquitous** [comment]
Hi @weaintplastic, thanks for reporting the problem!

I've just released [v0.13.3](https://github.com/ybiquitous/bem-ts/releases/tag/v0.13.3) to fix it. 🎉

**Comment by weaintplastic** [comment]
Hey @ybiquitous. Thanks so much for the quick fix. I can confirm its working in environments that support esmodules now. I also run this in test environments that require commonjs. 

I've created a PR that adds this line: https://github.com/ybiquitous/bem-ts/pull/895

**Comment by ybiquitous** [comment]
@weaintplastic Thank you for taking the time and submitting the pull request. I've just released [v0.13.4](https://github.com/ybiquitous/bem-ts/releases/tag/v0.13.4)! 🎉

**Comment by weaintplastic** [comment]
Thanks you so much. I can confirm it's working again. 🙌
